src/user_config_prompts.ts

```diff
@@ -14,7 +14,7 @@ function prompt(context: vs.ExtensionContext, key: string, prompt: () => Thenabl
 	//context.globalState.update(stateKey, undefined);
 
 	// If we've not prompted the user with this question before...
-	if (context.globalState.get(stateKey, false) !== true) {
+	if (context.globalState.get(stateKey) !== true) {
```

Comment:
:memo: The default value was unnecessary here, because `undefined` and `false` have the same result when compared strictly to `true`.